i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-firewalls-fortinet-fortigate-snmp.md

```diff
@@ -2,36 +2,1048 @@
+| --redis-db               | Set Redis database index.                                                                                                                                                                                                                     |
+| --failback-file          | Failback on a local file if Redis connection fails.                                                                                                                                                                                           |
+| --memexpiration          | Time to keep data in seconds (default: 86400).                                                                                                                                                                                                |
+| --statefile-dir          | Define the cache directory (default: '/var/lib/centreon/centplugins').                                                                                                                                                                        |
+| --statefile-suffix       | Define a suffix to customize the statefile name (default: '').                                                                                                                                                                                |
+| --statefile-concat-cwd   | If used with the '--statefile-dir' option, the latter's value will be used as a sub-directory of the current working directory. Useful on Windows when the plugin is compiled, as the file system and permissions are different from Linux.   |
+| --statefile-format       | Define the format used to store the cache. Available formats: 'dumper', 'storable', 'json' (default).                                                                                                                                         |
+| --statefile-key          | Define the key to encrypt/decrypt the cache.                                                                                                                                                                                                  |
+| --statefile-cipher       | Define the cipher algorithm to encrypt the cache (default: 'AES').                                                                                                                                                                            |
+| --filter-vdomain         | Filter by virtual domain name (can be a regexp).                                                                                                                                                                                              |
+| --add-traffic            | Add traffic usage by virtual domain.                                                                                                                                                                                                          |
```

Comment:
add?

Reply:
Without this option, you only monitor the state of the interface. With it, you collect the traffic statistics.